package.json

```diff
@@ -219,6 +219,8 @@
     "socket.io-client": "^4.7.5",
     "sqs-consumer": "^10.3.0",
     "sqs-consumer-v5": "npm:sqs-consumer@^5.7.0",
+    "square-calc": "^3.1.0",
```

Comment:
https://www.npmjs.com/package/square-calc
The Github Url is missing on the npm page.

And the README looks outdated. Could you please update the README and explaining:

square-calc v2 CJS
square-calc v3 ESM